ruleset/decoders/0140-kernel_decoders.xml

```diff
@@ -97,7 +97,7 @@ Example:
 <decoder name="iptables-OpenWRT">
    <parent>kernel</parent>
    <type>firewall</type>
-   <prematch>^[\d+.\d+] \S+\.*IN=</prematch>
+   <prematch>^[\d+.\d+] \S*\(\.*IN=</prematch>
```

Comment:
According to criteria tag order should be `decoder, type, parent, program_name, prematch, regex, order`